hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/OMConfigKeys.java

```diff
@@ -199,6 +199,14 @@ public final class OMConfigKeys {
       "ozone.om.ratis.log.appender.queue.byte-limit";
   public static final String
       OZONE_OM_RATIS_LOG_APPENDER_QUEUE_BYTE_LIMIT_DEFAULT = "32MB";
+
+  // OM Ratis Pending Write configurations
+  public static final String OZONE_OM_RATIS_PENDING_WRITE_NUM_LIMIT = "ozone.om.ratis.pending.write.num-limit";
+  public static final int OZONE_OM_RATIS_PENDING_WRITE_NUM_LIMIT_DEFAULT = 4096;
```

Comment:
Let's use `element-limit` instead of `num-limit` to standardize with Datanode `DatanodeRatisServerConfig`.

Also we can use `ozone.om.ratis.server` and instead of `ozone.om.ratis.pending` to standardize with the other existing `ozone.om.ratis.server` configuration. 

`ozone.om.ratis.pending.write.num-limit` -> `ozone.om.ratis.server.write.element-limit`